packages/frontend-2/components/settings/workspaces/Security.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <section>
+    <div class="md:max-w-xl md:mx-auto pb-6 md:pb-0">
+      <SettingsSectionHeader title="Security" text="Be secure 😤" />
+      <div>
+        <SettingsSectionHeader title="Workspace Domains" subheading />
+        <div
```

Comment:
It's just one button in this div, I think all the flex/gap classes are not needed?